templates/api-index.js

```diff
@@ -49,6 +49,36 @@ const handleError = (err) => {
   };
 }
 
+const scrub = (data) => {
+  if (!data || (typeof data !== 'object' && !Array.isArray(data))) return data;
+  const secrets = ["x_api_key", "x-api-key", "access_token", "access-token", "authorization", "api_key", "apiKey", "accessToken", "token", "password", "key"];
+  if (Array.isArray(data)) {
+    const temp = []
+    for (const item of data) {
+      if (Array.isArray(item) || typeof item === 'object') {
+        temp.push(scrub(item));
+      } else {
+        temp.push(item);
+      }
+    }
+    return temp;
+  }
+  else {
+    const temp = {};
+    for (const key of Object.keys(data)) {
+      if (Array.isArray(data[key]) || typeof data[key] === 'object') {
+        temp[key] = scrub(data[key]);
+      } else if (secrets.map(e => e.toLowerCase()).includes(key.toLowerCase())) {
```

Comment:
Let's just manually lowercase the secrets array. Then this can just become:
```suggestion
      } else if (secrets.includes(key.toLowerCase())) {
```